CMakeLists.txt

```diff
@@ -13,8 +13,8 @@ FetchContent_Declare(AndroidExtensions
     GIT_REPOSITORY https://github.com/BabylonJS/AndroidExtensions.git
     GIT_TAG 66520bff9b57030b67894a4934d18ad7e161ba6f)
 FetchContent_Declare(arcana.cpp
-    GIT_REPOSITORY https://github.com/microsoft/arcana.cpp.git
-    GIT_TAG c726dbe58713eda65bfb139c257093c43479b894)
+    GIT_REPOSITORY https://github.com/ryantrem/arcana.cpp.git
```

Comment:
TODO: Point to the real repo once https://github.com/microsoft/arcana.cpp/pull/51 is merged.